### PR TITLE
Fixes to activity display in case timeline

### PR DESCRIPTION
**Pull request by agh1**
Two issues when editing a case timeline:

- options for reference activities were not displaying per [CRM-20478](https://issues.civicrm.org/jira/browse/CRM-20478)
- activity types and statuses were displaying names, not labels--they weren't translated at all

### CODE REVIEW

File: ang/crmCaseType.js [hunk 1]
```diff
@@ -391,5 +397,6 @@
         });
     };
   });
+  // console.log($scope.)
```
Comment (eileenmcnaughton) [inline ang/crmCaseType.js]: kill this

**Comment by eileenmcnaughton** [comment]
@colemanw one for you to review?

File: ang/crmCaseType.js (hunk 1 above)
Comment (agh1) [inline ang/crmCaseType.js]: gone!

**Comment by agh1** [comment]
also noticed that "Newest" and "Oldest" were never translated

**Comment by seamuslee001** [comment]
@agh1 Andrew test failures look legit

**Comment by agh1** [comment]
@seamuslee001 thanks--the test was missing the activity type data for two of the activity types it used.  (My changes look up the activity type to find the label.)

**Comment by colemanw** [comment]
Code looks good, tested it out and was able to reproduce the bug & confirm that this fixes it.

**Pull request merged**